ci/check.sh

```diff
@@ -44,10 +44,9 @@ if [[ "$1" == "--check" ]]; then
 fi
 
 banner "Executing in conda environment ${CONDA_DEFAULT_ENV} in directory ${root}"
```

Comment:
This is preexisting, feel free to ignore. Looks like somewhere along the way this either got copypasta'd from another repo that actually defined `root` or the `root` definition was removed. Either way, `root` doesn't reference anything.

Reply:
thanks for catching!